presto-plugin-toolkit/src/main/java/com/facebook/presto/plugin/base/PluginClassLoader.java

```diff
@@ -26,7 +26,7 @@
 
 import static java.util.Objects.requireNonNull;
 
-class PluginClassLoader
+public class PluginClassLoader
```

Comment:
I don't think we need the refactor here any more. Can we move PluginClassLoader and PluginDiscovery back to their original locations? And revert the changes to presto-function-server